src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -632,6 +635,10 @@ def ignore_stack_frame(stack_frame, symbolized):
   # properly cross-platform.
   normalized_stack_frame = stack_frame.replace('\\', '/')
 
+  # Ignore frames that exactly match the app name.
+  if environment.get_value('APP_NAME') == normalized_stack_frame:
```

Comment:
can you move "environment.get_value('APP_NAME')" into a local and then also add a null check

if app_name and app_name == normalized_stack_frame ?

Reply:
Any reason for this? It should work fine with None.